### PR TITLE
convert components to use new core type vars

### PR DESCRIPTION
**Pull request by jaketrent**
Didn't remove old vars.  After this, is it time to do that? 

Changes according to this srategery:

## Font size

- [x] add 1000: 56px
- [x] add 1100: 72px
- [x] add 1200: 88px
- [x] fontSizeGigantic -> (unused)
- [x] fontSizeJumbo ->  Figma design change
  - [x] avatar - 
- [x] fontSizeXXLarge -> Figma design change
  - [x] avatar
  - [x] datawell
- [x] fontSizeXLarge -> (unused)
- [x] fontSizeLarge -> fontSize600
- [x] fontSizeMedium -> fontSize400
- [x] 16px -> fontSizeBase
- [x] fontSizeSmall -> fontSize200
- [x] fontSizeXSmall -> fontSize100

## Letter spacing

- [x] letterSpacingGigantic -> (unused)
- [x] letterSpacingJumbo -> (unused)
- [x] letterSpacingXXLarge -> (unused)
- [x] letterSpacingXLarge -> (unused)
- [x] letterSpacingLarge -> Figma design change
  - [x] emptystate
  - [x] errors
- [x] letterSpacingMedium -> (unused)
- [x] letterSpacingSmall -> Figma design change
  - [x] emptystate
  - [x] errors
- [x] letterSpacingXSmall -> (unused)

## Font weight

- [x] fontWeightBlack -> fontWeight900
- [x] fontWeightXBold -> fontWeight800
- [x] fontWeightBold -> fontWeight700
- [x] fontWeightDemiBold -> fontWeightStrong
- [x] fontWeightMedium -> fontWeight500
- [x] fontWeightRegular -> fontWeightDefault
- [x] fontWeightBook -> fontWeightDefault
- [x] fontWeightLight -> fontWeight300
- [x] fontWeightXLight -> fontWeight200
- [x] fontWeightThin -> fontWeight100

## Line height

- [x] lineHeightTight -> (remains)
- [x] lineHeightStandard -> (remains)
- [x] lineHeightExtra -> (remains)
- [x] lineHeightHuge -> (remains)

### CODE REVIEW

File: commit.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+for FILE in $(git diff --name-only); do
+  PACKAGE=`echo $FILE | sed "s|packages/\(.*\)/src.*|\1|"`
+  git add "packages/$PACKAGE"
+  git commit -m "refactor($PACKAGE): direct conversion to new core names for fontWeight"
+done
```
Comment (danethurber) [inline commit.sh:5]: did you mean to add this to vcs

**Review by danethurber — COMMENTED** [review]
everything looks good to me. i'm not approving since I think @sweeting should do that - but 👍  from me

File: commit.sh (hunk 1 above)
Comment (jaketrent) [reply commit.sh:5]: lol, nope.  Will rm. Thanks.

File: packages/table/src/css/index.ts [hunk 2]
```diff
@@ -110,7 +110,7 @@ export default {
     textAlign: 'left',
 
     '& > div': {
-      fontWeight: type.fontWeightBold,
+      fontWeight: type.fontWeight700,
```
Comment (sweeting) [inline packages/table/src/css/index.ts:113]:
```suggestion
      fontWeight: type.fontWeightStrong,
```

File: packages/verticaltabs/src/css/index.ts [hunk 3]
```diff
@@ -209,7 +209,7 @@ export default {
     alignItems: 'center',
     display: 'flex',
     fontSize: '11px',
-    fontWeight: type.fontWeightBold,
+    fontWeight: type.fontWeight700,
```
Comment (sweeting) [inline packages/verticaltabs/src/css/index.ts:212]:
```suggestion
    fontWeight: type.fontWeightStrong,
```

**Review by sweeting — CHANGES_REQUESTED** [review]
- Made a few adjustments to match some design changes in Figma.
- Changed all references to `fontWeight700` to `fontWeightStrong`

**Comment by jaketrent** [comment]
Hey @sweeting , I took all your changes and committed them.  Please re-review.

**Review by sweeting — APPROVED** [review]
👍

**Pull request merged**